data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,174 @@
+				accept
+			
+mission "Acorn Delights: Part 5"
+	landing
+	name "Acorns for Anoo"
+	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."
+	source Darkcloak
+	passengers 1
+	cargo "acorn cookies" 3
+	to offer
+		has "Acorn Delights: Part 4: done"
+	on visit
+		dialog "You forgot Eeeya or some of her mountains of cookies. Maybe they're in one of your escorts?"
+	on offer
+		"salary: Acorn Delights Dividends" = 500
+		payment 30000
+		log `Helped the famous Hai chef Eeeya find her granddaughter, a trauma surgeon named Anoo. Both joined the Unfettered hoping they can, "set them on a more productive path." Received shares of Acorn Delights stock which pay 500 credits a day in dividends.`
+		conversation
+			`You're ushered quickly from the landing platform to a nearby hospital. Unfettered Hai nearby are muttering in their language, but you keep hearing the Hai word for, "cookies" as they eye Eeeya. Along the way, you notice an odd contrast of squalor and advanced machinery. It seems the Unfettered value function and progress, rather than aesthetics and comfort like their southern brethren.`
+			`	As you rise through the main tower of the hospital, you're surprised at how advanced it is. Despite their willingness to die in battle, the Unfettered seem to value the lives of the survivors. Perhaps this is to heal them quickly to fight again?`
+			`	You reach a high floor, filled with personal offices. Eeeya takes you to a door at the end of a long hallway.`
+			choice
+				`	(Open the door.)`
+			`	 After the door opens, Eeeya and Anoo run to each other's arms. A few minutes later, they begin to speak. Eventually the conversation turns to Anoo's new occupation.`
+			`	"Why did you come here, Anoo? The Unfettered hate us."`
+			`	"They don't hate US, grandma. They hate what we've become: frivolous, uninventive, and self-important. Our race is fading into history through a slow death of mediocrity. Most Unfettered see warfare as our only salvation.`
+			`	"We think of them as mindless brutes, but they're not! They have scientists, doctors, musicians, and writers. Our resources dwarf theirs, but they thrive with what little they have. Soon the Unfettered will have many more planets and almost unlimited resources. The era of scarcity will be over, and they'll have the power to choose their future.`
+			`	"If enough good people join the Unfettered, we can change their minds; set them on a more productive path. They'll need good doctors."`
+			`	Eeeya smiles at her granddaughter, "They'll need a good cook too."`
+			choice
+				`	"I'm glad to have reunited you."`
+				`	"I'd like to be paid now."`
+			`	Eeeya turns to you and says, "I've found my new home, Captain <last>. I suppose payment is in order. I'm the owner of Eeeya's Acorn Delights, a luxury baked goods company based on Allhome.`
```

Comment:
It does seem a little odd Eeeya is only choosing to to tell you this now when she's been traveling with you this long (especially if she's well-known enough for the flight controller to know who she is).

Reply:
Two thoughts come to mind: 
1) Maybe there should be an opportunity for the pilot to ask her why the ATC recognized her?
2) Could set up a branch here: If the player takes the earlier opportunity to ask her, then they find out then and this spot here can skip the explanation. But if the player doesn't ask, then they get this explanation.